kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
Is this an auto-generated file? If it is not, I suggest you add hash instead of SNAPSHOT. we can add generating this file to the generator along with containerizing the generation.